src/components/DashCard/DashCard.tsx

```diff
@@ -15,11 +16,15 @@ const Card = styled.article`
 export const DashCard: React.FC<IDashCard> = ({
   title,
   content,
+  icon,
 }: IDashCard): React.ReactElement => {
   return (
     <Card className="dashcard-container">
       <h2 className="dashcard-title">{title}</h2>
-      <h1 className="dashcard-content">{content}</h1>
+      <h1 className="dashcard-content">
+        {icon}
```

Comment:
Here you should probably just be passing the icon needed then render the font awesome component inside here. 

<FontAwesomeIcon className="dashboard-icon" icon={props.icon} />

Reply:
updated in latest commit